addons/website_event_exhibitor/static/src/js/event_sponsor_search.js

```diff
@@ -10,13 +10,30 @@ publicWidget.registry.websiteEventSearchSponsor = publicWidget.Widget.extend({
         'click .o_search_tag .btn': '_onTagRemove',
         'click .o_dropdown_reset_tags': '_onTagReset',
         'change .o_wevent_event_tags_form input': '_onTagAdd',
+        'input .o_wevent_event_search_box': '_onClearInput',
+        'keyup .o_wevent_event_search_box': '_onKeyUp',
```

Comment:
Maybe something slightly more specific? _onClearSearchInput and _onSearchKeyUp? :)